components/concourse-github-resource/assets/in

```diff
@@ -103,49 +92,51 @@ EOF
 } | tr -d '\n'
 }
 
-function approvals() {
-  local commit_sha=$1
-  local org=$2
-  local repo=$3
-  local token=$4
-  local api_response
-
-  api_response=$( \
-    curl --header 'Content-Type: application/json' \
-         --header "Authorization: bearer ${token}" \
-         --data "{ \"query\": \"$(build_query "${org}" "${repo}")\"}" \
-         "https://api.github.com/graphql" \
-  )
-
-  echo "${api_response}" \
-    | jq --raw-output .data.organization.repository.pullRequests.nodes[] \
-    | jq --raw-output ".reviews.nodes[] | select(.commit.oid==\"${commit_sha}\")" \
```

Comment:
it seems we no longer directly check that the commit associated with a review is the commit sha. I think that's fine (we rely on github to dismiss reviews when people push commits) but I just wanted to check that this was deliberate?

Reply:
I believe it is deliberate.

Reply:
This is a good question—I would expect the approvals to be dismissed but I'm not sure we've actually verified this is the case.

Reply:
They are dismissed for PRs onto the master branch, it's in the repo settings.

Reply:
...although we have reason to suspect they aren't *always* dismissed :(